playwright/utils/mockApi.ts

```diff
@@ -8,7 +8,7 @@ import type {
 } from '../../src/vendors/nexus/index'
 import { StringifiedBigInt } from '../../src/types/StringifiedBigInt'
 
-export async function mockApi(context: BrowserContext | Page, balance: StringifiedBigInt) {
+export async function mockApi(context: BrowserContext | Page, balance: number | StringifiedBigInt) {
```

Comment:
oh I did not update all mockApi usage. this is minor and not related to this PR, but `Account['available']` would be more accurate. can bump it once this is one is merged if that works for you.